### ISSUE TITLE
Text Size instead of Text Scale

### ISSUE THREAD

**Issue by lovato** [jan-molak/jenkins-build-monitor-plugin#254]
What happens is that our builds grow along the day, like everyone else.
Then, if I set a proper font size for our TV, at the start of the day, showing lets say 4 builds, at the end of the day, after new 20 builds, nobody can read things anymore, since the font size decreases as much as new builds came in.

How can I set a font size and stick to it, no matter the number of builds the dashboard shows up?

Tks
Marco

**Comment by jan-molak** [comment]
Hey Marco! From what I've seen so far, in most cases the number of builds that the Build Monitor displays remains relatively constant throughout the day (usually when people have 20 projects that they want to display, they'll have 20 projects displayed on screen all the time).

Can you tell me a bit more about your scenario? What makes the new projects appear on screen throughout the day? A screenshot of how your display looks in the morning (with few projects) and in the evening (with 20 projects) would also be helpful!

Best,
Jan

**Comment by lovato** [comment]
Sure! This is a typical morning dashboard.

![image](https://cloud.githubusercontent.com/assets/1213774/18167485/395ff434-7027-11e6-9685-63fb3472d448.png)

For each repo, we use 3 jobs, one to build from master, one to build for develop, and one to build a Pull Request prior to accept a merge.

And this is my config (perhaps this will explain some aspects):

![image](https://cloud.githubusercontent.com/assets/1213774/18167553/97c28fdc-7027-11e6-866e-b68736b0a6c0.png)

Explaining, I only want to show lastest builds (someone from other geography may commit something at mi night, so I wanna see it on my morning), and I want to have all problematic builds on the top, permanently. I have 156 jobs, so I dont care OK builds from two days ago or older, they are history. They are solved.

Thats why, along the day, the number of builds do increase, up to 4 times.
Below I did a simulation, increasing 16h to 160h to look back.
This is how I would see it at the end of the day:

![image](https://cloud.githubusercontent.com/assets/1213774/18167749/a8f1aada-7028-11e6-9bd9-897f3c5d5e0c.png)

Best!

**Comment by lovato** [comment]
This is a FRAME on a HTML. What I just noticed is that at a certain width, a scroll bar do show up, and the font stops getting smaller. Using this width, the plugin tries to render everything on the same HTML area. Perhaps this can solve everything...

**Comment by jan-molak** [comment]
Hey @lovato and thanks for the thorough explanation and the examples! Could you please elaborate a bit more on your last message?

> This is a FRAME on a HTML.

Do you mean that you display the Build Monitor _within a frame_, or am I misunderstanding?

> What I just noticed is that at a certain width, a scroll bar do show up, and the font stops getting smaller. Using this width, the plugin tries to render everything on the same HTML area. 

Yes, Build Monitor will show the scroll bars if it can't fit all the items within the screen, and will only decrease the font size [until a threshold](https://github.com/jan-molak/jenkins-build-monitor-plugin/blob/495d38f597ea1e923927f2638dfcd10837353704/build-monitor-plugin/src/main/webapp/scripts/controllers.js#L41).

> Perhaps this can solve everything...

Not sure if I follow? What are you thinking?

**Comment by lovato** [comment]
Yes, it do show up inside a frame, since the other frame I list all PullRequests opened.
I said about solving because if I enlarge a little more the width for Build Monitor, it will probably looks better.

Will try it and return back to you.

**Comment by jan-molak** [comment]
Cool, thanks. I designed Build Monitor to be displayed on large displays and radiate the information across the office floor. Since this is its primary purpose, the support for embedding within other dashboards is not really there ...
